api/src/test/java/org/openmrs/PersonNameTest.java

```diff
@@ -78,6 +78,60 @@ public void newInstance_shouldCopyEveryPropertyOfGivenPersonName() throws Except
 	}
 
 	/**
+	 * @see PersonName#newInstance(PersonName)
+	 */
+	@Test
+	@Verifies(value = "should trim leading and trailing whitespaces on each name attribute", method = "newInstance(PersonName)")
+	public void newInstance_shouldTrimLeadingAndTrailingWhitespacesOnEachNameAttribute() throws Exception {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method names should not contain underscores](https://www.codacy.com/app/openmrs/openmrs-core/file/2318747982/issues/source?bid=3204099&fileBranchId=3204128#l85)